emulators/libkrun/Portfile

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+# Upstream build source (until more official): https://github.com/slp/homebrew-krunkit/blob/main/Formula/libkrun-efi.rb
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo_fetch 1.0
+
+github.setup        containers libkrun 1.9.5 v
+github.tarball_from archive
+
+revision            0
+
+categories          emulators
+license             Apache-2
+maintainers         {@tsujp wz.ht:jc+macports} openmaintainer
+
+description         Dynamic library providing Virtualization-based process isolation capabilities
+long_description    Dynamic library that allows programs to easily acquire the \
+                    ability to run processes in a partially isolated environment \
+                    using Hypervisor.framework on macOS.
+
+# Only arm64 as libkrun only supports Hypervisor.framework on arm64.
+supported_archs     arm64
+# TODO: A way to specify Hypervisor.framework is installed? Is it always installed if XCode is? In that case (as MP requires XCode IIRC then no need for extra checks).
```

Comment:
It's always there, either in Xcode, or in the Command Line Tools, which are required by MacPorts:

/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/System/Library/Frameworks/Hypervisor.framework/Versions/A

Reply:
Thanks for confirmation!

Reply:
then the comment can be removed?

Reply:
Yes.